src/pages/about.js

```diff
@@ -7,11 +7,83 @@ const AboutPage = ({data, location}) => {
 
   return (
     <Layout>
-      <SEO title="About :: What does it take to decarbonize your state?" />
+      <SEO title="About :: Decarbonize My State" />
       <div className="container">
-        <div className="border-bottom">
-          <h1 id="main" className="py-2">Overview</h1>
-          <p>About this app</p>
+        <div className="about">
+          <h1 id="main border-bottom" className="py-4">About Decarbonize My State</h1>
+          <p>We have 28 years to reduce worldwide emissions to zero to avoid the worst impacts of climate change. But how do we get there and what can I do to help?</p>
```

Comment:
We should make the number of years dynamic so this doesn't have to be manually updated